.github/release-drafter.yml

```diff
@@ -16,7 +16,7 @@ categories:
   - 'support'
   - 'dependencies'
 category-template: '### $TITLE'
-change-template: '- $TITLE ([#$NUMBER](https://github.com/ryu-sato/awesome-backup/pull/$NUMBER)) [@$AUTHOR](https://github.com/$AUTHOR)'
+change-template: '- $TITLE (#$NUMBER) @$AUTHOR'
```

Comment:
Use changelog only for releases and pull requests; pull request IDs and authors do not need to be links.
If you use the changelog in a markdown file such as CHANGELOG.md, you should consider using a link.